.sclng/metadata.toml

```diff
@@ -0,0 +1,8 @@
+dependencies = []
+description = "Service monitoring actively containers running on a server"
+destination_server = ["app-hosting-node"]
+flags = ["deployed-by-chef", "tools"]
+languages = ["go"]
+owner = "leo@scalingo.com"
+team = "IST"
+version = "1.0.2"
```

Comment:
If there is a version number here, the README should be updated, the section "Release a New Version"

Reply:
This version comes from asgard, cf: https://github.com/Scalingo/asgard-cli/blob/master/metadata/init.go#L70